src/analysis/max_min_driving_force.jl

```diff
@@ -114,7 +153,7 @@ function max_min_driving_force(
 
     # add the relative bounds
     for ((mid1, mid2), val) in concentration_ratios
-        idxs = indexin([mid1, mid2], mids)
+        idxs = indexin([mid1, mid2], metabolites(model))
```

Comment:
probably a good to go but having an index would be safer ;D

Reply:
Not sure I know what you mean?

Reply:
indexin here is potentially `O(n_metabolites^3)`  (# of all possible ratios * size). Let's leave it hangin for now but this may eventually get terribly slow.

Reply:
okay, will leave a TODO